apps/admin_api/test/admin_api/v1/controllers/admin_auth/api_key_controller_test.exs

```diff
@@ -178,6 +185,36 @@ defmodule AdminAPI.V1.AdminAuth.APIKeyControllerTest do
     end
   end
 
+  describe "/api_key.enable_or_disable" do
```

Comment:
Can you add a test that disabling it twice doesn't make it enabled?

Just in case someone turns it into a toggle in the future.